src/db/database.providers.ts

```diff
@@ -90,7 +90,7 @@ export const databaseProviders = [
         InterviewExperienceModel,
       ]);
 
-      // await sequelize.sync({ force: true });
+      await sequelize.sync({ alter: true });
```

Comment:
Although this also works, this line is anyways kind of hackish and will have to be written in migrations separately.